dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -426,6 +428,7 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 										front.config.pageId,
 										collection.displayName,
 										numBannerAdsInserted,
+										index === 0,
```

Comment:
Minor but could this be extracted to a const?

Reply:
Have another PR coming. Will make this change there!